doc/ChangeLog

```diff
@@ -1,4 +1,89 @@
 ===============================================================
+Tag name: ctsm5.3.071
+Originator(s): jinmuluo (Jinmu Luo, Cornell University)
+Date: Tue 19 Aug 2025 06:19:28 PM MDT
+One-line Summary: New vertical movement scheme for soil nitrate
+
+Purpose and description of changes
+----------------------------------
+
+ This tag introduces a new vertical movement scheme for soil nitrate but sets its use to .false. by default. Details about the new scheme in the PR #2992.
+
+Significant changes to scientifically-supported configurations
+--------------------------------------------------------------
+
+Does this tag change answers significantly for any of the following physics configurations?
+(Details of any changes will be given in the "Answer changes" section below.)
+
+    [Put an [X] in the box for any configuration with significant answer changes.]
+
+[ ] clm6_0
+
+[ ] clm5_0
+
+[ ] ctsm5_0-nwp
+
+[ ] clm4_5
+
+
+Bugs fixed
+----------
+List of CTSM issues fixed (include CTSM Issue # and description) [one per line]:
+ No corresponding github issue
+
+Notes of particular relevance for users
+---------------------------------------
+Changes to CTSM's user interface (e.g., new/renamed XML or namelist variables):
+ use_nvmovement can be set by user
+
+Changes made to namelist defaults (e.g., changed parameter values):
+ use_nvmovement defaults to .false.
+
+Changes to documentation:
```

Comment:
I would ask that @jinmuluo provide updates to the documentation.  This doesn't have to be a requirement for the PR to be merged, but maybe it's something at @samsrabin can help with when Jinmu is in Boulder this fall?